core/embed/rust/src/ui/model_tt/theme/mod.rs

```diff
@@ -158,21 +158,21 @@ pub const fn label_progress() -> TextStyle {
 }
 
```

Comment:
Can you please also fix `theme/bootloader.rs` and `component/result.rs`?

Reply:
regarding `theme/bootloader.rs` is that necessary since we linked `FONT_BOLD` to `_upper` font as well?

Reply:
Anyway, fix here eb495f2b6c6d2a72d05661132cc248ed4c045d9c